pycytominer/cyto_utils/load.py

```diff
@@ -41,20 +81,25 @@ def load_profiles(profiles):
     Return
     ------
     pandas DataFrame of profiles
+
+    Raises
+    ------
+    FileNotFoundError
+        Raised if the provided
     """
     if not isinstance(profiles, pd.DataFrame):
-        try:
-            delim = infer_delim(profiles)
-            profiles = pd.read_csv(profiles, sep=delim)
-        except FileNotFoundError:
-            raise FileNotFoundError(f"{profiles} profile file not found")
+
+        if is_path_a_parquet_file(profiles):
+            return pd.read_parquet(profiles, engine="pyarrow")
+        
+        delim = infer_delim(profiles)
+        profiles = pd.read_csv(profiles, sep=delim)
+
     return profiles
 
 
 def load_platemap(platemap, add_metadata_id=True):
     """
-    Unless a dataframe is provided, load the given platemap dataframe from path or string
```

Comment:
Why delete this docstring?

Reply:
hmmm... It doesn't appear deleted on my end :thinking: